operator/pkg/ingress/ingress_dedicated.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2019-2020 Authors of Cilium
```

Comment:
Please remove the year (likely incorrect anyway)

Reply:
thanks good spot :100: